### ISSUE TITLE
Horizontal heat transfer connections do not work in CFAST 7.2.4

### ISSUE THREAD

**Issue by rpeacoc** [firemodels/cfast#1236]
With a namelist input file, horizontal heat transfer connections do not work in CFAST 7.2.4.  The same input file as a spreadsheet input works fine.

**Comment by rpeacoc** [comment]
Here's an input file that demonstrates the problem.  Compartment 2 should heat up.

```
&HEAD VERSION = 7300, TITLE = 'CFAST Simulation' /
 
!! Scenario Configuration 
&TIME SIMULATION = 8000 PRINT = 60 SMOKEVIEW = 60 SPREADSHEET = 60 / 
&INIT PRESSURE = 101325 RELATIVE_HUMIDITY = 50 INTERIOR_TEMPERATURE = 20 EXTERIOR_TEMPERATURE = 20 /
&MISC  MAX_TIME_STEP = 2 LOWER_OXYGEN_LIMIT = 0.15 / 
 
!! Material Properties 
&MATL ID = 'STEEL3/8' MATERIAL = '"Steel, Plain Carbon (3/8 in)"', 
      CONDUCTIVITY = 48 DENSITY = 7854 SPECIFIC_HEAT = 0.559, THICKNESS = 0.0095 EMISSIVITY = 0.9 /
 
!! Compartments 
&COMP ID = 'Comp 1'
      DEPTH = 3 HEIGHT = 3 WIDTH = 3 CEILING_MATL_ID = 'OFF' WALL_MATL_ID = 'STEEL3/8' FLOOR_MATL_ID = 'OFF'
      ORIGIN = 0, 0, 0 GRID = 50, 50, 50 /
&COMP ID = 'Comp 2'
      DEPTH = 3 HEIGHT = 3 WIDTH = 3 CEILING_MATL_ID = 'OFF' WALL_MATL_ID = 'STEEL3/8' FLOOR_MATL_ID = 'OFF'
      ORIGIN = 3, 0, 0 GRID = 50, 50, 50 /
 
!! Wall Vents
&VENT TYPE = 'WALL' ID = 'WallVent_1' COMP_IDS = 'Comp 1' 'OUTSIDE'  TOP = 2, BOTTOM = 0, WIDTH = 1
      FACE = 'LEFT' OFFSET = 1 /
 
!! Fires 
&FIRE ID = 'New Fire'  COMP_ID = 'Comp 1', FIRE_ID = 'New Fire'  LOCATION = 1.5, 1.5 / 
&CHEM ID = 'New Fire' CARBON = 1 CHLORINE = 0 HYDROGEN = 4 NITROGEN = 0 OXYGEN = 0 HEAT_OF_COMBUSTION = 50000 RADIATIVE_FRACTION = 0.35 / 
&TABL ID = 'New Fire' LABELS = 'TIME', 'HRR' , 'HEIGHT' , 'AREA' , 'CO_YIELD' , 'SOOT_YIELD' , 'HCN_YIELD' , 'HCL_YIELD' , 'TRACE_YIELD'  /
&TABL ID = 'New Fire', DATA = 0, 1000, 0, 0.3, 0.06, 0.1, 0, 0, 0 /
&TABL ID = 'New Fire', DATA = 8000, 1000, 0, 0.3, 0.06, 0.1, 0, 0, 0 /
 
!! Surface Connections
&CONN TYPE = 'WALL' COMP_ID = 'Comp 1'  COMP_IDS =   'Comp 2' 'OUTSIDE'
      F = 0.25 0.75  /
&CONN TYPE = 'WALL' COMP_ID = 'Comp 2'  COMP_IDS =   'Comp 1' 'OUTSIDE'
      F = 0.25 0.75  /
```

**Comment by rpeacoc** [comment]
Fixed in 8d049b72036862dab2719def29993410f49ed16b
